src/app/[locale]/(app)/(nav)/overview/_components/TopChannels.tsx

```diff
@@ -7,14 +7,20 @@ import useTopChannelsData from "~/hooks/data/use-top-channels-data";
 import { useTranslation } from "~/i18n/client";
 import { firstCharFromUnicode } from "~/utils";
 import { iconColor } from "~/utils/discord";
+import useScroller from "~/hooks/use-scroller";
+import { useRef } from "react";
 
 export default function TopChannels() {
   const { t } = useTranslation();
   const data = useTopChannelsData().getData({});
+  const cardsRef = useRef<HTMLDivElement>(null);
```

Comment:
@florian-lefebvre is it better to return ref from the hook itself rather than us creating it manually?

Reply:
I think it would be better to use useScroller instead of the ScrollArea, and make sure it runs only when orientation is horizontal. Wdyt?